### ISSUE TITLE
No longer working

### ISSUE THREAD

**Issue by livejamie** [tyler-england/listeddit#1]
Is this project on hold?

**Comment by tyler-england** [comment]
Yes. As far as the code, I haven't worked on anything past the scope of music (Spotify). I would like for it to be functional with respect to movies/books/etc. though, so feel free to contribute! As far as the bot itself, I wasn't sure whether I'd restart it after Reddit's controversial API changes. It is back up and running now though.

**Comment by magrubertone** [comment]
I don't think this is working anymore is it? Honestly don't mind if it doesn't work for anything other than music, I think that's it's best use case for sure.
